pkg/tests/testdata/expected/support-2.out

```diff
@@ -34,19 +34,16 @@ INSERT 0 10
 ANALYZE;
 ANALYZE
 EXPLAIN (costs off) SELECT time, value, jsonb(labels), val(namespace_id) FROM cpu_usage WHERE labels ? ('namespace' !== 'dev' ) ORDER BY time, series_id LIMIT 5;
-                                       QUERY PLAN                                       
-----------------------------------------------------------------------------------------
+                                       QUERY PLAN                                        
+-----------------------------------------------------------------------------------------
  Limit
-   InitPlan 1 (returns $0)
```

Comment:
why is the initPlan still not showing up here?

Reply:
Fixed!